plugin/mlsdbprovider.h

```diff
@@ -128,7 +128,10 @@ private Q_SLOTS:
     void startPositioningIfNeeded();
     void stopPositioningIfNeeded();
     void setStatus(Status status);
-    void getEnabled(bool *positioningEnabled, bool *cellPositioningEnabled, bool *onlinePositioningEnabled);
+    void getEnabled(bool *positioningEnabled, bool *cellPositioningEnabled,
+                    bool *onlinePositioningEnabled, bool *onlineDataAllowed,
+                    bool *cellDataAllowed, bool *cellSignalStrengthAllowed,
+                    bool *wlanDataAllowed, bool *wlanSignalStrengthAllowed);
```

Comment:
Yikes. I am starting to wish some struct/object was passed which provided more type and order safety.

Reply:
it's used exactly once, I don't think it's an issue, personally.  Also the signal strength stuff has been removed, which makes it smaller now at least..

Reply:
True its an internal implementation detail.